### DISCUSSION TITLE
PDF电子签章验证出错，RSA验签时只支持小于128K的文件

### DISCUSSION THREAD

**Discussion by songjinlian** [Leon406/ToolsFx#253] [Q&A]
PDF电子签章验证出错，RSA验签时只支持小于128K的文件。PDF文件大小为4M多。出错截图如下：
![图片](https://github.com/Leon406/ToolsFx/assets/101912268/20e5c64c-a065-48e6-8386-ff45d7ce7aea)

**Comment by Leon406** [comment]
不支持二进制文件
